cloud/azure/azure_rm_deployment.py

```diff
@@ -42,7 +42,7 @@
       - In incremental mode, resources are deployed without deleting existing resources that are not included in the template. 
         In complete mode resources are deployed and existing resources in the resource group not included in the template are deleted.
```

Comment:
Please add some notes here that the default hast changed: e.g. `- The default changed in version 2.3 from C(complete) to C(incremental).`